ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -464,6 +471,7 @@ private[state] object Conversions {
           buildStatus(
             Code.INVALID_ARGUMENT,
             s"Disputed: ${rejection.getDetails}",
+            Map(GrpcStatuses.DefiniteAnswerKey -> "true"),
```

Comment:
Looks like we always just set this key so we can move it into the `buildStatus` as a bool value

Reply:
I had an impression that in the next story, we may want to add the "structured metadata" here.

Reply:
I would still split them, so for the metadata just add a new field to the build status so that there's a clear split between them at least in the builder

Reply:
Yes, I'm going to add it here, so I would leave it as is.

Reply:
Ok